### PR TITLE
[MIG] trp_external_user

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by hbrunn** [comment]
@StefanRijnhart ping

File: trp_external_user/README.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+partner (even if global write access on partners are not granted to the external
+users group in this module).
+
+This module also disables the default assignment of the 'user' and 'partner manager'
+groups to new users.
+
+Caution
+=======
+
+This module is incompatible with the sahe module.
```
Comment (StefanRijnhart) [inline trp_external_user/README.rst]: sa**l**e?

**Comment by StefanRijnhart** [comment]
Would you consider migrating this functionality to the standard 'portal' group?

**Comment by hbrunn** [comment]
@StefanRijnhart I lift on the share module now, that seems nicer to me

**Pull request merged**